secret-detector/src/main/java/com/expedia/www/haystack/pipes/secretDetector/SpringConfig.java

```diff
@@ -33,17 +33,13 @@
 import com.expedia.www.haystack.pipes.commons.kafka.KafkaConfigurationProvider;
 import com.expedia.www.haystack.pipes.commons.kafka.KafkaStreamStarter;
 import com.expedia.www.haystack.pipes.commons.serialization.SerdeFactory;
-import com.expedia.www.haystack.pipes.secretDetector.actions.EmailerDetectedAction;
-import com.expedia.www.haystack.pipes.secretDetector.actions.EmailerDetectedActionFactory;
-import com.expedia.www.haystack.pipes.secretDetector.actions.SenderImpl;
+import com.expedia.www.haystack.pipes.secretDetector.actions.*;
```

Comment:
Can we revert this class completely for now?

Reply:
To support the new way of configuration and also give backward compatibility, we need to add this